components/atom/colorPanel/README.md

```diff
@@ -0,0 +1,25 @@
+# AtomColorPanel
+
+> A color panel is a container with a color on its background.
+
+## Installation
+
+```sh
+$ npm install @schibstedspain/sui-atom-colorPanel --save
```

Comment:
Do we use uppercase for package names ???

Reply:
nope, its the autogenerated readme that does this and I didn't check it. will be fixed